Assets/__Scripts/MapEditor/UI/SongSpeedController.cs

```diff
@@ -29,17 +29,17 @@ private void Update()
     public void OnDecreaseSongSpeed(InputAction.CallbackContext context)
     {
         if (!context.performed) return;
-        songSpeed--;
-        if (songSpeed < 1) songSpeed = 1;
+        songSpeed -= (float)(Settings.Instance.SongSpeedChangeAmount/10.0f);
```

Comment:
Float cast is unnecessary as this float division. Also fix the whitespace:
```suggestion
        songSpeed -= Settings.Instance.SongSpeedChangeAmount / 10.0f;
```